src/i18n/it.js

```diff
@@ -0,0 +1,43 @@
+export default {
+    header: {
+        slogan: 'Il futuro dello sviluppo web.'
+    },
+    callToAction: {
+        start: 'Per cominciare',
+        join: 'Unisciti alla chat'
+    },
+    nav: {
+        code: 'Code',
+        cloud: 'Cloud',
+        docs: 'Documentazione'
+    },
+    bio: {
+        first: Architettura non bloccante, basata sugli eventi grazie alla libreria <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> di Apple che garantisce <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">alte prestazioni</a>.',
+        second: 'Scritto in <a target="_blank" href="https://swift.org">Swift</a>, il linguaggio di programmazione che è potente e facile da imparare.',
+        third: 'Espressivo, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">orientato ai protocolli</a> con un occhio di riguardo alla type-safety e alla mantenibilità.'
+    },
+    contribute: {
+        title: 'Contribuisci!',
+        options: {
+            code: {
+                description: 'Invia idee, segnala bug, ed esegui merge su GitHub!'
+            },
+            chat: {
+                title: 'Unisciti alla chat',
+                description: 'Aiuta i nuovi arrivati ad iniziare ad usare Vapor.'
+            },
+            donate: {
+                title: 'Diventa uno sponsor',
```

Comment:
Should use capital S in 'sponsor'

Reply:
Why? Shouldn't we use capitalization only for brands and people?

Reply:
To give more importance to possible future sponsor, the English version also uses the capital there

Reply:
I'll do it then, even if I think it's not necessary (and I wouldn't definitely do it if it were my site …)